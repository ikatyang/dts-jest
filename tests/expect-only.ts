declare const expect_only: () => {result: any};

// @dts-jest 1: expect
expect_only();

// @dts-jest:only 2: expect-only
expect_only();

// @dts-jest 3: expect
expect_only();
