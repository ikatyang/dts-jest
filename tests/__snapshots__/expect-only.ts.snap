// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2: expect-only 1`] = `"{ result: any; }"`;
